client/src/main/java/io/pravega/client/stream/impl/ReaderGroupImpl.java

```diff
@@ -174,13 +205,18 @@ public void resetReadersToCheckpoint(Checkpoint checkpoint) {
     }
 
     @Override
-    public void updateConfig(ReaderGroupConfig config, Set<String> streamNames) {
+    public void resetReaderGroup(ReaderGroupConfig config) {
         @Cleanup
         StateSynchronizer<ReaderGroupState> synchronizer = createSynchronizer();
-        Map<Segment, Long> segments = getSegmentsForStreams(streamNames);
+        Map<Segment, Long> segments = getSegmentsForStreams(config);
         synchronizer.updateStateUnconditionally(new ReaderGroupStateInit(config, segments));
     }
 
+    @Override
+    public void updateConfig(ReaderGroupConfig config) {
+      resetReaderGroup(config);
```

Comment:
I thought that `updateConfig` would have different semantics from `resetReaderGroup`?